db/schema.rb

```diff
@@ -309,6 +318,8 @@
     t.string "key", null: false
     t.datetime "disabled_at"
     t.boolean "can_upload_documents", default: false, null: false
+    t.uuid "category_id"
```

Comment:
the assumption here is that there is 1 category per location - it would be worth double-checking this with @Nimphal 's expert in the MoJ

Reply:
Jade has confirmed with the PMU that each location has only one category - and that's what the location category will be used for so I think it makes sense to go with this structure until that assertion changes. That's preferable to adding a has_many relationship just in case we might need it I think. We should go with what we're told really rather than try to second guess.

Reply:
MoJ expert has confirmed there can be a mixture of prisoner categories in a single prison. E.g. a Cat B prison can hold Cat B and Cat C prisoners. But it should not hold prisoners with a rating higher than the prison - e.g. a Cat B prison should not hold Cat A prisoners.